bucket/hcloud.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "1.21.0",
+    "description": "hcloud: Command-line interface for Hetzner Cloud",
+    "homepage": "https://github.com/hetznercloud/cli",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/hetznercloud/cli/blob/master/LICENSE"
+    },

```

When it's a license identified by SPDX no link is needed

```suggestion
    "license": "MIT",
```